src/qc_data_preparation/helpers/dataset.py

```diff
@@ -90,3 +95,143 @@ def _exists(self) -> bool:
 
     def _describe(self) -> Dict[str, Any]:
         return dict(filepath=self._filepath)
+
+
+class PTModelDataset(AbstractDataSet):
+    def __init__(
+        self,
+        filepath: str,
+        load_args: Dict[str, Any] = None,
+        save_args: Dict[str, Any] = None,
+    ) -> None:
+        self._tmp_prefix = "kedro_pytorch_tmp"  # temp prefix pattern
+        self._filepath = filepath
+        self._load_args = load_args
+        self._save_args = save_args
+        _fs_args = {}
+        protocol, path = get_protocol_and_path(filepath)
+        if protocol == "file":
+            _fs_args.setdefault("auto_mkdir", True)
+
+        self._protocol = protocol
```

Comment:
Consider to directly assign here and discard the unused part.
```suggestion
        self.protocol, _ = get_protocol_and_path(filepath)
        if self.protocol == "file":
            _fs_args.setdefault("auto_mkdir", True)l
```